seatunnel-api/src/main/java/org/apache/seatunnel/api/configuration/util/OptionRule.java

```diff
@@ -146,43 +153,49 @@ public Builder required(Option<?>... options) {
         /**
          * Exclusive options, only one of the options needs to be configured.
          */
-        public Builder exclusive(Option<?>... options) {
+        public Builder exclusive(@NonNull Option<?>... options) {
             if (options.length <= 1) {
                 throw new OptionValidationException("The number of exclusive options must be greater than 1.");
             }
             for (Option<?> option : options) {
+                verifyDuplicate(option, "ExclusiveOption");
                 verifyRequiredOptionDefaultValue(option);
             }
             this.requiredOptions.add(RequiredOption.ExclusiveRequiredOptions.of(options));
             return this;
         }
 
-        public Builder exclusive(RequiredOption.ExclusiveRequiredOptions exclusiveRequiredOptions) {
+        public Builder exclusive(@NonNull RequiredOption.ExclusiveRequiredOptions exclusiveRequiredOptions) {
+            exclusiveRequiredOptions.getExclusiveOptions().forEach(option -> {
+                verifyDuplicate(option, "ExclusiveOption");
+            });
             this.requiredOptions.add(exclusiveRequiredOptions);
             return this;
         }
 
-        /**
-         * Conditional options, These options are required if the {@link Option} == expectValue.
-         */
-        public <T> Builder conditional(Option<T> option, T expectValue, Option<?>... requiredOptions) {
-            return conditional(Condition.of(option, expectValue), requiredOptions);
-        }
```

Comment:
IMO, The tool method of a single expected value need not be deleted.

Reply:
> IMO, The tool method of a single expected value need not be deleted.

Ok.